content/about/xmpp-standards-foundation.md

```diff
@@ -19,8 +19,9 @@ If you have an interest in XMPP standards, it is straightforward to [become a Me
 The Board of Directors of the XMPP Standards Foundation oversees the business affairs of the organisation. As elected by the XSF membership, the Board of Directors for 2020-2021 consists of the following individuals:
 
 - __Arc Riley__ has been contributing to the free software community for over two decades, and maintains a large number of freely licensed XMPP packages. He lead for Python XML-SIG, mentor students working on XMPP-related projects and frequently presents on XMPP (lightning talks, poster sessions, workshops, etc).
-- __JC Brand__ is the creator of [Converse.js XMPP Messenger](https://conversejs.org) and has worked as an XMPP consultant and developer since 2013. You can find him on the [Fediverse](https://mastodon.xyz/jcbrand) and chat with him on [XMPP](xmpp:jc@opkode.com).
+- __Edward Maurer__ is working in engineering automation (energy sector, non-XMPP) and is volunteering in XMPP since more than five years already. Since 2020 he is thriving the [XSF Communication Team](https://xmpp.org/about/xsf/comm-team/). Besides successfully supporting the XMPP Community and its projects he is co-author of [XMPP Providers](https://providers.xmpp.net/). Find the contact and details in the [XMPP wiki profile](https://wiki.xmpp.org/web/User:Echolon).
 - __Matthew Wild__ is a founder of the [Prosody XMPP server](http://prosody.im/) project and a number of [smaller XMPP projects](http://code.matthewwild.co.uk/). Matthew has been a member of the XMPP Council since 2009, has authored and co-authored various [XEPs](/extensions/) and helps maintain the [Jabber.org IM service](http://jabber.org/).
+- __Peter Saint-Andre__ is involved with the Jabber/XMPP community since late 1999. Up until 2014 I served as Council member, Editor, Secretary, Treasurer,  Executive Director, etc. I also (co-)authored a lot of specs, both XEPs and RFCs. Peter was very active in the Internet Engineering Task Force (IETF) as a spec author, working group chair, member of the Internet Engineering Steering Group (IESG) and chair of the RFC Series Oversight Committee (RSOC) and more. He also advises tech companies but mainly serves a non-tech startup in the business training space.
```

Comment:
```suggestion
- __Peter Saint-Andre__ is a semi-retired technologist who has been involved with the Jabber/XMPP community since late 1999, including many years as Executive Director.  Although he currently advises several tech companies, he is mainly focused on a startup that offers training to early- and mid-career managers.
```